CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+**2.6.0** (5 Aug 2015)
+- add `time_event` method
+- update snippet to load mixpanel correctly from cdn when ﬁle is opened locally
```

Comment:
digression @zakj we could have a git hook that requires text files to contain only ASCII.
We probably have some unit-tests today that use raw Unicode, but they should all be changed to use `\u` and so on, anyway (so that we only have to trust "does Python handle \u sequences correctly", as opposed to "does every text editor and source code tool in the world handle non-ASCII bytes correctly").